keras_nlp/models/electra/electra_presets.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Electra model presets and configurations."""
+
+backbone_presets = {
+    "google/electra-base-generator": {
+        "metadata": {
+            "description": (
+                "Base Electra generator model from Google."
+                "ELECTRA models are trained to distinguish 'real' input tokens vs 'fake' input "
+                "tokens generated by another neural network, similar to the discriminator of a GAN.",
+            ),
+            "params": "33576960 ",
+            "official_name": "Electra",
+            "path": "electra",
+            "model_card": "https://huggingface.co/google/electra-base-generator",
+        },
+        "config": {
+            "vocab_size": 30522,
+            "embedding_dim": 768,
+            "num_layers": 12,
+            "num_heads": 4,
+            "hidden_dim": 256,
+            "intermediate_dim": 1024,
+            "dropout": 0.1,
+            "max_sequence_length": 512,
+        },
+        "preprocessor_config": {
+            "lowercase": False,
+        },
+        # TODO: Upload weights on GCS.
+        "weights_url": "https://storage.googleapis.com/pranav-keras/electra-base-generator/model.weights.h5",
```

Comment:
We actually just moved all our weights over to Kaggle. https://github.com/keras-team/keras-nlp/releases/tag/v0.7.0

This will make it easier to upload models long term, but let me get back to you next week on exact steps for upload. If you have a kaggle username, could you reply here with it?

Reply:
[kaggle](https://www.kaggle.com/pranavprajapati16) here is my kaggle username